postcss.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Ich würde es bevorzugen, wenn wir diese Konfiguration mit in die webpack-Config packen, wie hier im ersten Codebeispiel beschrieben:
https://github.com/webpack-contrib/postcss-loader